src/workflows/templates/schemas.ts

```diff
@@ -64,18 +64,24 @@ export function buildCreateRequestSchema(
 ): OpenAPIV3.SchemaObject {
   return {
     type: "object",
-    required: ["id", "type"],
+    required: ["data"],
     properties: {
-      id: {
-        type: "string",
-        format: "uuid",
-      },
-      type: {},
-      attributes: {
-        $ref: `#/components/schemas/${titleResourceName}CreateAttributes`,
+      data: {
+        type: "object",
+        required: ["id", "type"],
```

Comment:
We generally don't want to allow clients to specify IDs. We might need to allow it in certain circumstances but normally we'll want server-assigned IDs.
```suggestion
        required: ["type"],
```